src/app/components/ManualEntry.svelte

```diff
@@ -87,47 +159,78 @@
   input:focus {
     background-color: var(--lite-white);
   }
+
+  .save-zone {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 1%;
+  }
 </style>
 
-<div class="language__sources-manual-entry">
+<form class="language__sources-manual-entry" on:submit|preventDefault>
   <div class="language__sources-manual-entry-tablename">
     <h4>Table Name</h4>
-    <input data-cy="manual-entry-input-tablename" />
+    <input
+      type="text"
+      bind:value={tableData.name}
+      required
+      data-cy="manual-entry-input-tablename" />
   </div>
 
-  <table>
+  <table id="manual-entry-table">
     <thead>
-      <th>Word</th>
-      <th>Count</th>
-      <th>Action</th>
+      {#each columns as column (column.name)}
+        <th>{column.label}</th>
+      {/each}
```

Comment:
This is what I meant. I think it's best to just hard code the headers here.
```suggestion
      <th> Word </th>
      <th> Count </th>
      <th> Action </th>
```